mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -229,6 +229,7 @@ func Provider() *schema.Provider {
 			"google_compute_network_endpoint_group":            dataSourceGoogleComputeNetworkEndpointGroup(),
 			"google_compute_node_types":                        dataSourceGoogleComputeNodeTypes(),
 			"google_compute_regions":                           dataSourceGoogleComputeRegions(),
+			"google_compute_region_network_endpoint_group":		dataSourceGoogleComputeRegionNetworkEndpointGroup(),
```

Comment:
you have tabs between the datasource name and method where other lines have spaces